doc/source/configuration/monitoring.rst

```diff
@@ -78,3 +93,31 @@ on the overcloud hosts:
 
 SMART reporting should now be enabled along with a Prometheus alert for
 unhealthy disks and a Grafana dashboard called ``Hardware Overview``.
+
+Alertmanager and Slack
+======================
+
+StackHPC Kayobe configuration comes bundled with an array of alerts but does not
+enable any receivers for notifications by default. Various receivers can be
+configured for Alertmanager. Slack is currently the most common.
+
+To set up alerts for a Slack channel, you will need a webhook. `Create a new app
+<https://api.slack.com/apps/new>`__ in the workspace you want to add alerts to.
+From the Features page, toggle Activate incoming webhooks on. Click Add new
+webhook to workspace. Pick a channel that the app will post to, then click
```

Comment:
We could clarify here that one App is fine for two channels, so with the bundled config you could add the webhooks to both the critical and non-critical channels. Sort of covered below I guess.

Reply:
Good spot, I've clarified the point